contact.html

```diff
@@ -52,17 +52,22 @@
         </h1>
       <div class="field">
         <label>Name</label>
-        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="name" placeholder="Name" required />
       </div>
       <div class="field">
         <label>E-mail</label>
-        <input type="text" name="email" placeholder="E-mail">
+        <input type="text" name="email" placeholder="E-mail" required />
```

Comment:
Use "email" as the type for form validation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input